snuba/request/schema.py

```diff
@@ -169,6 +169,8 @@ def generate_template(self) -> Any:
             # first replica, so should only be used when absolutely necessary.
             "consistent": {"type": "boolean", "default": False},
             "debug": {"type": "boolean", "default": False},
+            # Maps to the transaction name of the Sentry API call
+            "parent_api": {"type": "string", "default": "<unknown>"},
```

Comment:
Same concern I had above. This should not be a setting. It is not supposed to be usable by the client to instruct snuba how to run the query. 
I would even consider making it a top level element.